.github/workflows/molecule.yml

```diff
@@ -0,0 +1,42 @@
+---
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          path: "${{ github.repository }}"
+      - name: molecule
+        uses: robertdebock/molecule-action@2.6.3
+        with:
+          command: lint
+  test:
+    needs:
+      - lint
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
```

Comment:
@Chithrak07 since we [advertise this role to work for any red hat / debian flavor](https://github.com/AcroMedia/ansible-role-ssh-user/blob/master/meta/main.yml#L7), lets add a couple more to this matrix: 
```
- geerlingguy/docker-centos7-ansible
- geerlingguy/docker-centos8-ansible
```

Reply:
Done